packages/teleport/src/services/mfa/types.ts

```diff
@@ -15,4 +31,8 @@ export type AddNewTotpDeviceRequest = {
 export type AddNewHardwareDeviceRequest = {
   tokenId: string;
   deviceName: string;
+  deviceUsage?: DeviceUsage;
 };
+
+export type DeviceType = 'totp' | 'webauthn';
+export type DeviceUsage = 'passwordless' | 'mfa';
```

Comment:
This name is a bit confusing imo, why not something like `AuthOption`?

Reply:
`DeviceUsage` is sim to how it is defined in the back, it describes how a device will be used (a device can be used as passwordless, or as a multi factor e.g password + device)